bootstrap/service_test.go

```diff
@@ -34,12 +34,20 @@ const (
 	channelsNum  = 3
 )
 
-var config = bootstrap.Config{
-	ExternalID:  "external_id",
-	ExternalKey: "external_key",
-	MFChannels:  []string{"1"},
-	Content:     "config",
-}
+var (
+	channel = bootstrap.Channel{
+		ID:       "1",
```

Comment:
Not UUIDv4?

Reply:
@drasko You're right, they are. It was easier for me to use incremental IDs in tests, but I'm planning to update tests as a part of #540. Is it OK to leave it as it is for now since tests update will take me a lot of time?

Reply:
Yes, OK.